Popular_Models_Guide/Llava1.5/llava_trtllm_guide.md

```diff
@@ -0,0 +1,222 @@
+```
+
+
+> Optional: You can check test the output of the model with `run.py`
+> located in the same llama examples folder.
+>
+>   ```bash
+>    python3 /tensorrtllm_backend/tensorrt_llm/examples/multimodal/run.py --max_new_tokens 30 --hf_model_dir ${HF_LLAVA_MODEL} --visual_engine_dir ${ENGINE_DIR} --llm_engine_dir ${ENGINE_DIR} --decoder_llm --input_text "Question: which city is this? Answer:"
+>    ```
+> You should expect the following response:
+> ```
+> [TensorRT-LLM] TensorRT-LLM version: 0.9.0
+> ...
+> [06/18/2024-01:02:24] [TRT-LLM] [I] ---------------------------------------------------------
+> [06/18/2024-01:02:24] [TRT-LLM] [I]
+> [Q] Question: which city is this? Answer:
+> [06/18/2024-01:02:24] [TRT-LLM] [I]
+> [A] ['Singapore']
+> [06/18/2024-01:02:24] [TRT-LLM] [I] Generated 1 tokens
+> [06/18/2024-01:02:24] [TRT-LLM] [I] ---------------------------------------------------------
+> ```
+
+### Serving with Triton
+
+The last step is to set up a Triton model repository. For this tutorial,
+we provide all necessary Triton related files under `model_repository/`.
+You simply need to provide TensorRT-LLM engine location in its `config.pbtxt`:
+
+```bash
+FILL_TEMPLATE_SCRIPT=/tensorrtllm_backend/tools/fill_template.py
+python3 ${FILL_TEMPLATE_SCRIPT} -i /tutorials/Popular_Models_Guide/Llava1.5/model_repository/tensorrt_llm/config.pbtxt engine_dir:${ENGINE_DIR}
+```
+
+3.  Launch Tritonserver
+
+Use the [launch_triton_server.py](https://github.com/triton-inference-server/tensorrtllm_backend/blob/release/0.5.0/scripts/launch_triton_server.py) script. This launches multiple instances of `tritonserver` with MPI.
+```bash
+export TRT_ENGINE_LOCATION="/engines/llava1.5/visual_encoder.engine"
+export HF_LOCATION=/llava-1.5-7b-hf
+python3 /tensorrtllm_backend/scripts/launch_triton_server.py --world_size=<world size of the engine> --model_repo=/opt/tritonserver/inflight_batcher_llm
+```
+> You should expect the following response:
+> ```
+> ...
+> I0503 22:01:25.210518 1175 grpc_server.cc:2463] Started GRPCInferenceService at 0.0.0.0:8001
+> I0503 22:01:25.211612 1175 http_server.cc:4692] Started HTTPService at 0.0.0.0:8000
+> I0503 22:01:25.254914 1175 http_server.cc:362] Started Metrics Service at 0.0.0.0:8002
+> ```
+
+To stop Triton Server inside the container, run:
+```bash
+pkill tritonserver
+```
+
+### Send an inference request
+
+You can test the results of the run with:
+1. The [multi_modal_client.py](tutorials/Popular_Models_Guide/Llava1.5/multi_modal_client.py) script.
+
+```bash
+# Using the SDK container as an example
+docker run --rm -it --net host --shm-size=2g \
+    --ulimit memlock=-1 --ulimit stack=67108864 --gpus all \
+    -v /path/to/tutorials:/tutorials
+    nvcr.io/nvidia/tritonserver:<xx.yy>-py3-sdk
+
+python3 python multi_modal_client.py --prompt "Describe the picture." --image_url "http://images.cocodataset.org/test2017/000000155781.jpg" --max-tokens=15
```

Comment:
```suggestion
python3 multi_modal_client.py --prompt "Describe the picture." --image_url "http://i
mages.cocodataset.org/test2017/000000155781.jpg" --max-tokens=15
```
I think it would be great if specifying the path for the multi_modal_client.py script `/tutorials/Popular_Models_Guide/Llava1.5`.